Formula/fd.rb

```diff
@@ -0,0 +1,18 @@
+class Fd < Formula
+  desc "Simple, fast and user-friendly alternative to find."
+  homepage "https://github.com/sharkdp/fd"
+  url "https://github.com/sharkdp/fd/archive/v1.0.0.tar.gz"
+  sha256 "711ecfd34640fb4d840567231992102aa55da471aaafbffb7c2b76073872092e"
+  head "https://github.com/sharkdp/fd.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
+    bin.install "target/release/fd"
+  end
+
+  test do
+    system "#{bin}/fd"
```

Comment:
To expand on @ilovezfs comment: This is the core tap so we have more strict rules than e.g. if you were writting a formula for [your own tap](http://docs.brew.sh/How-to-Create-and-Maintain-a-Tap.html).